web/docs/advanced/links.md

```diff
@@ -40,6 +40,44 @@ export const TaskList = () => {
 }
 ```
 
+### Catch-all Routes
+
+If a route path ends with a `/*` pattern (also known as [splat](https://reactrouter.com/en/main/route/route#splats)), you can use the `Link` component like this:
```

Comment:
```suggestion
If a route path ends with a `/*` pattern (also known as a [splat](https://reactrouter.com/en/main/route/route#splats)), you can use the `Link` component like this:
```
Also, is this only allowed at the route's end? Can't it be in the middle as well?